Exercise2.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Linq1
+{
+    public static class Exercise2
+    {
+        // Given a sequence of words, get rid of any that don't have the character 'e' in them,
+        // then sort the remaining words alphabetically, then return the following phrase using
+        // only the final word in the resulting sequence:
+        //    -> "The last word is <word>"
+        // If there are no words with the character 'e' in them, then return null.
+        //
+        // TRY to do it all using only LINQ statements. No loops or if statements.
+        public static string GetTheLastWord(this IEnumerable<string> words)
+        {
+            // using Lambda expression Syntax
+            // return $"The last word is {words.Where(st => st.Contains("e")).OrderBy(st => st).Select(st => st).ToArray().LastOrDefault()}";
+            string g = words.Where(st => st.Contains("e")).OrderBy(st => st).Select(st => st).ToArray().LastOrDefault();
+            if (g == null)
+            {
+                return "null";
```

Comment:
they mean the actual null not the string "null"

Reply:
for the first attempt i use the actual null to solve the problem , but the compiler for website need to print a null as string to let me pass the exercise ...